.github/workflows/pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: macos-latest
```

Comment:
```suggestion
    # Build the fonts on macOS because we use the default overlap removal library
    # booleanOperations, which unfortunately inherits
    # https://sourceforge.net/p/polyclipping/bugs/195/ that manifests for certain
    # glyphs like `shade` and `dkshade`. The contour order makes no visual
    # difference, but is important for TrueType hinting, where instructions work on
    # point indices.  The font was originally build on macOS and then hinted in VTT,
    # so we have to use macOS to not potentially invalidate the hinting.
    runs-on: macos-latest
```